modules/ROOT/pages/_partials/task-add-api-instance.adoc

```diff
@@ -72,14 +72,22 @@ CloudHub proxy includes a CPU with 0.1 vCores and a memory of 500 MB, by default
 ... Select the *Version* for Mule runtime.
 ... Select the *Java version*.
 .. Enter a *Proxy app name*.
-* *Hybrid:* Select Hybrid if you have a Mule runtime running on an on-premises server that you want to use.
+
+* *Hybrid:* Select Hybrid if you want to use a Mule runtime running on an on-premises server that is registered in Runtime Manager.
 See xref:runtime-manager::servers-create.adoc[Register a server in Runtime Manager] for more information.
 +
+.. Select a target from the list or click **Add server**.
+.. Enter a *Proxy app name*.
+
+* *Self-managed Server:* Select self-managed server if you want to create a JAR file to deploy an API proxy to a Mule runtime running on an on-premises server that is not registered in Runtime Manager. For more information, see xref:mule-runtime::mule-deployment-model.adoc[].
```

Comment:
Should self-managed server be capitalized in the "Select ..." sentence? (How do they refer to it in the UI?)